packages/contracts/src/ubiquistick/UbiquiStick.sol

```diff
@@ -103,7 +103,7 @@ contract UbiquiStick is
         }
     }
 
-    function random() private view returns (uint256) {
+    function _random() private view returns (uint256) {
```

Comment:
we're relying on **block.prevrandao** @rndquu to avoid compiling warnings as foundry.toml is shanghai

??

Reply:
I don't understand the question

Reply:
Yes, your PR sets it to [prevrandao](https://github.com/ubiquity/ubiquity-dollar/pull/704/files#diff-8dd9ecc8e1f48ed00e1ff605d96ff7d2f9e865c9d2b609b5b55fbe5b764ca2b0R111) but this PR merges into the `development` branch which uses the "old" `difficulty`. I still can't get what you mean